src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -81,6 +82,7 @@ const VideoDetailPage = () => {
         video: videoData?.videoUrl,
         courseKey: videoData?.courseKey,
         title: videoData?.courseTitle,
+        video_usage_key: videoData?.videoUsageKey,
```

Comment:
[nit] Should the key for this value be camelCased as `videoUsageKey` to keep in parity with `courseKey`